src/js/components/molecules/list-section/list-section.ts

```diff
@@ -32,8 +33,28 @@ export default class ListSection extends Vue implements IListSection {
     rowIndex: number = 0;
     incement: number = 1;
 
+    $emitPublic;
     viewportUtil: ViewportUtil = new ViewportUtil();
 
+    get clickEventType() {
+        return Events.click;
+    }
+
+    get clickEventData() {
+        if (this.expand && this.expand.clickEvent) {
+            const { clickEvent } = this.expand;
+
+            return {
+                dataType: clickEvent.name,
+                data: {
+                    ... clickEvent.data,
+                    pageSlug: window.location.pathname,
```

Comment:
This url should come from the current route (this.$router) instead of the global window object

Reply:
Is this.$router available in designsystem?